netci.groovy

```diff
@@ -142,7 +142,7 @@ def CreateLinuxBuildTasks = { machine, configTag, linuxBranch, nonDefaultTaskSet
                     }
                 }
 
-                def archivalString = "BuildLinux/build.log"
+                def archivalString = "Build/clang_build/x64_${buildFlag}/build.log"
```

Comment:
>${buildFlag} [](start = 60, length = 12)

I think this is incorrect -- values will be:
release -> ""
debug -> "--debug"
test -> "--test-build"

You probably meant to use `${buildType}` which is one of ['debug', 'test', 'release'] (see line 118)